# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/warn-no-phys-cpuid.patch
# Copyright (C) 2021 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Improve diagnostics, warn once when no phys acpi cpu could be found.
 
--- linux-5.14/drivers/acpi/acpi_processor.c	2021-08-30 00:04:50.000000000 +0200
+++ linux-5.14/drivers/acpi/acpi_processor.c	2021-09-26 18:44:44.540473555 +0200
@@ -277,7 +279,7 @@
 	pr->phys_id = acpi_get_phys_id(pr->handle, device_declaration,
 					pr->acpi_id);
 	if (invalid_phys_cpuid(pr->phys_id))
-		dev_dbg(&device->dev, "Failed to get CPU physical ID.\n");
+		dev_warn_once(&device->dev, "Failed to get CPU physical ID.\n");
 
 	pr->id = acpi_map_cpuid(pr->phys_id, pr->acpi_id);
 	if (!cpu0_initialized && !acpi_has_cpu_in_madt()) {
